JavaFXUserInterface/src/main/java/SystemInfoController.java

```diff
@@ -31,6 +31,37 @@
 import java.util.ResourceBundle;
 
 public class SystemInfoController implements Initializable {
+    private static final String S1_COL = "s1";
+    private static final String S2_COL = "s2";
+    private static final String S3_COL = "s3";
+    private static final String S4_COL = "s4";
+    private static final String S5_COL = "s5";
+    private static final String S6_COL = "s6";
+    private static final String S7_COL = "s7";
+    private static final String S8_COL = "s8";
+    private static final String S9_COL = "s9";
+    private static final String S10_COL = "s10";
+    private static final String S11_COL = "s11";
+    private static final String S12_COL = "s12";
+    private static final String S13_COL = "s13";
+    private static final String S14_COL = "s14";
+    private static final String S15_COL = "s15";
+    private static final String S16_COL = "s16";
+    private static final String S17_COL = "s17";
+    private static final String S18_COL = "s18";
+    private static final String S19_COL = "s19";
+    private static final String S20_COL = "s20";
+    private static final String S21_COL = "s21";
+    private static final String S22_COL = "s22";
+    private static final String S23_COL = "s23";
+    private static final String S24_COL = "s24";
+    private static final String S25_COL = "s25";
+    private static final String S26_COL = "s26";
+    private final String CYCLE_COL = "Cycle";
+    private final String OP1_COL = "OP1";
+    private final String OP2_COL = "OP2";
+    private final String OP3_COL = "OP3";
```

Comment:
This is not dynamic enough, the amount and names of the attributes have to come from the database, not hard coded